src/portals/hope/features/claims/claim-details/SubclaimPaymentOrders/components/SubclaimPaymentOrderForm.tsx

```diff
@@ -155,9 +155,19 @@ const SubclaimPaymentOrderForm: React.FC<{
   }, [setValue, watch])
 
   useEffect(() => {
+    if (!watch('reference')) {
```

Comment:
Feels like we don't want it to be `hedvig`
```suggestion
    if (!watch('reference') && member?.memberId) {
```